### PR TITLE
doc updates

### PR DESCRIPTION
**Pull request by bsuryadevara**
Removed outdated imports and fixed minor errors in the docs notebooks.

### CODE REVIEW

**Review by ajschmidt8 — COMMENTED** [review]
Not sure I'm the best person to review notebook updates. I don't see anything overtly wrong with these changes though. I'll defer to others for an official approval.

File: docs/source/intro-clx-dga.ipynb [hunk 1]
```diff
@@ -127,7 +127,9 @@
      "name": "stderr",
```
Comment (efajardo-nv) [inline docs/source/intro-clx-dga.ipynb:127]: <p>prediction is different here. is this expected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (bsuryadevara) [reply docs/source/intro-clx-dga.ipynb:127]: <p>Because we're demonstrating model training rather than actual training, this is to be expected.</p>

**Review by efajardo-nv — APPROVED** [review]

**Comment by bsuryadevara** [comment]
@gpucibot merge

**Pull request merged**